src/pages/Settings/TwoStepVerificationPage.vue

```diff
@@ -20,16 +20,66 @@
 							name="auto-icon-on"
 						/>
 					</template>
+
 					<template v-slot:content>
-						<p>You'll be asked for a verification code when accessing you Kiva account.</p>
+						<div v-if="!isMfaActive">
+							<h2>{{ cardTitle }}</h2>
+							<p>{{ cardSubhead }}</p>
+						</div>
+
+						<div v-if="isMfaActive" class="section">
+							<h3 class="strong">
+								2-step verification is turned on
+							</h3>
+							<p>
+								We'll ask you for your password and verification code from your
+								mobile phone in order to log into your Kiva account.
+							</p>
+							<a
+								@click="checkLastLoginTime"
```

Comment:
This feels funny without an `href`. I'd recommend making it a` <kv-button class="text-link" ...`